appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java

```diff
@@ -63,7 +64,7 @@ public AssetList() {
       @Override
       public void onClick(ClickEvent event) {
         if (assetsFolder != null) {
-          FileUploadWizard uploader = new FileUploadWizard(assetsFolder);
+          DNDFileUploadWizard uploader = new DNDFileUploadWizard(assetsFolder);
```

Comment:
Yes, the normal FileUploadWizard is used to upload projects and in the Properties panel.